src/mimemessage.cpp

```diff
@@ -82,6 +84,25 @@ bool MimeMessage::write(QIODevice *device)
         return false;
     }
 
+    const auto dt = QDateTime::currentDateTime();
+    data = QByteArrayLiteral("Date: ") + QLocale::c().toString(dt, QStringLiteral("ddd, d MMM yyyy HH:mm:ss ")).toLatin1();
```

Comment:
Can you take a look at how I build Date headers in Cutelyst, it's possible that you can simplify this +0000 with '+0000 directly on the toString()